crates/telio-firewall/src/firewall.rs

```diff
@@ -364,6 +387,9 @@ impl StatefullFirewall {
             IpNextHeaderProtocols::Tcp => {
                 self.handle_outbound_tcp(&ip);
             }
+            IpNextHeaderProtocols::Icmp => {
```

Comment:
If I remember correctly, Next Header protocol for IPv6 ICMP is different from IPv4, so we should have a case for `IpNextHeaderProtocols::Icmpv6` (refernece: https://docs.rs/pnet/latest/pnet/packet/ip/IpNextHeaderProtocols/constant.Icmpv6.html)

Reply:
In Matas' PR he handles this case, but it's essentially just generics and calling the same function